include/astra_camera/astra_driver.h

```diff
@@ -101,9 +169,10 @@ class AstraDriver
   void initDevice();
 
   void advertiseROSTopics();
+  void closeROSTopics();
 
-  void imageConnectCb();
-  void depthConnectCb();
+  void imageConnectCb(bool grab_image_client_ = false);
+  void depthConnectCb(bool grab_image_client_ = false);
```

Comment:
Remove trailing underscore (non-member variable), and maybe use plural in image.